cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -2,11 +2,19 @@
 
 from . import utils
 
+# AUTOMATED, DO NOT CHANGE -- IF GENCODE UPDATED, DEPRECATED FEATURE FILTERING ALGORITHM WILL GO HERE.
+# fmt: off
+DEPRECATED_FEATURE_IDS = [
+]
+# fmt: on
+
 
 def migrate(input_file, output_file, collection_id, dataset_id):
     print(f"Converting {input_file} into {output_file}")
 
-    dataset = ad.read_h5ad(input_file)
+    dataset = ad.read_h5ad(input_file, backed="r")
+    if dataset.raw is not None and DEPRECATED_FEATURE_IDS:
+        dataset = dataset.to_memory()
```

Comment:
out of curiosity, what changed your mind to use backed="r" for onotlogy-only updates? won't it be slower and not save us on resources since we'll still provision based on loading in-memory worst case?

Reply:
answered in meeting: allows curators to test ontology-only changes locally without loading in-memory on their machines unnecessarily